flexget/plugins/filter/crossmatch.py

```diff
@@ -99,10 +102,17 @@ def entry_intersects(self, e1, e2, fields=None):
             log.trace('v1: %r' % v1)
             log.trace('v2: %r' % v2)
 
-            if v1 == v2:
-                common_fields.append(field)
+            if config.get('fuzzy', {}):
+                if v2 in v1:
```

Comment:
I wonder if this should check the other way too: `if v2 in v1 of v1 in v2:`. Not sure though...